bears/general/FilenameBear.py

```diff
@@ -71,6 +77,18 @@ def run(self, filename, file,
                      file_naming_convention)
             return
 
+        if not filename_without_extension.startswith(prefix):
+            yield Result.from_values(
+                origin=self,
+                message='Filename does not use the {} prefix.'.format(prefix),
+                file=filename)
+
+        if not filename_without_extension.endswith(suffix):
+            yield Result.from_values(
+                origin=self,
+                message='Filename does not use the {} suffix.'.format(suffix),
+                file=filename)
```

Comment:
oh we even need to merge it below with https://github.com/coala/coala-bears/pull/1836/commits/23360b43da416d3afae8573e646d24ecb83ae6fe#diff-8da09498fa5dc8583a01d5e2f76de875R98 :/